src/main/resources/edu/arizona/sista/reach/biogrammar/events_master.yml

```diff
@@ -106,8 +118,113 @@ rules:
       label: "Methylation"
       verbalTriggerLemma: "methylate"
       nominalTriggerLemma: "methylation"
+      triggerPrefix: "!word=/(?i)^de/"
+      priority: "5"
+
+
+  #
+  # "Reverse" Simple Events:
+
+  # Dephosphorylations
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Dephosphorylation"
+      actionFlow: "mkBioMention"
+      label: "Dephosphorylation"
+      verbalTriggerLemma: "de-?phosphorylate"
+      nominalTriggerLemma: "de-?phosphorylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Deubiquitination
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Deubiquitination"
+      actionFlow: "mkUbiquitination"
+      label: "Deubiquitination"
+      verbalTriggerLemma: "de-?ubiquitinate"
+      nominalTriggerLemma: "de-?ubiquitination"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Dehydroxylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Dehydroxylation"
+      actionFlow: "mkBioMention"
+      label: "Dehydroxylation"
+      verbalTriggerLemma: "de-?hydroxylate"
+      nominalTriggerLemma: "de-?hydroxylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
+
+  # Desumoylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Desumoylation"
+      actionFlow: "mkBioMention"
+      label: "Desumoylation"
+      verbalTriggerLemma: "de-?sumoylate"
+      nominalTriggerLemma: "de-?sumoylation"
+      triggerPrefix: "word=/(?i)^de/"
       priority: "5"
 
+  # Deglycosylation
+  - import: edu/arizona/sista/reach/biogrammar/events/simple-event_template.yml
+    vars:
+      eventName: "Deglycosylation"
+      actionFlow: "mkBioMention"
+      label: "Deglycosylation"
+      verbalTriggerLemma: "de-?glycosylate"
+      nominalTriggerLemma: "de-?glycosylation"
+      triggerPrefix: "word=/(?i)^de/"
+      priority: "5"
```

Comment:
No, you don't want to encode the prefix in two places.  The `verbalTriggerLemma` and `nominalTriggerLemma` should be identical to the corresponding additive event.  For example, "deglycosylation" and "glycosylation" should have the same `verbalTriggerLemma` and `nominalTriggerLemma`; the distinction can be made in the `triggerPrefix`.